usersync/chooser.go

```diff
@@ -34,13 +34,14 @@ func NewChooser(bidderSyncerLookup map[string]Syncer, biddersKnown map[string]st
 
 // Request specifies a user sync request.
 type Request struct {
-	Bidders        []string
-	Cooperative    Cooperative
-	Limit          int
-	Privacy        Privacy
-	SyncTypeFilter SyncTypeFilter
-	GPPSID         string
-	Debug          bool
+	Bidders           []string
+	Cooperative       Cooperative
+	Limit             int
+	Privacy           Privacy
+	SyncTypeFilter    SyncTypeFilter
+	GPPSID            string
+	Debug             bool
+	CookieDeprecation CookieDeprecation
```

Comment:
Why is this defined here? The user sync chooser does not care about cookie deprecation.

Reply:
removed